tests/src/test/java/alluxio/cli/GetConfKeyTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.PropertyKey;
+import alluxio.SystemOutRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+
+/**
+ * Tests for {@link GetConfKey}.
+ */
+public final class GetConfKeyTest {
+  private ByteArrayOutputStream mOutputStream = new ByteArrayOutputStream();
+
+  @Rule
+  public SystemOutRule mOutputStreamRule = new SystemOutRule(mOutputStream);
+
+  @Test
+  public void getConfKey() throws Exception {
```

Comment:
split into 3 tests. When possible it's better to have more short tests so that they can pass/fail independently and the method names serve as documentation

Reply:
Done.